src/gallium/drivers/lima/lima_tiling.c

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2011-2013 Luc Verhaegen <libv@skynet.be>
+ * Copyright (c) 2018 Alyssa Rosenzweig <alyssa@rosenzweig.io>
+ * Copyright (c) 2018 Lima Project
```

Comment:
You may sign your name as the feedback in drm-lima RFC "Lima Project" is not a real org.

Reply:
OK